tests/migrations/test_writer.py

```diff
@@ -321,6 +322,29 @@ class IntEnum(enum.IntEnum):
             "default=migrations.test_writer.IntEnum(1))"
         )
 
+    def test_serialize_uuid(self):
+        uuid_a = uuid.UUID('5c859437-d061-4847-b3f7-e6b78852f8c8')
```

Comment:
I would like to see that the round trip works too, e.g.:

    +        self.assertSerializedEqual(uuid.uuid1())
    +        self.assertSerializedEqual(uuid.uuid4())